jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/thread/ThreadTagParser.java

```diff
@@ -59,23 +60,25 @@ public void processLine(String source, int startingOffset) {
 		 * Tags with no attributes (or attributes to be ignored)
 		 */
 		addTag(IThreadTypes.T_1XMTHDINFO, null);
+		addTag(IThreadTypes.T_3XMJAVALTHREAD, null);
+		addTag(IThreadTypes.T_3XMHEAPALLOC, null);
```

Comment:
What is the value of these additions (and the explicit optional check for `3XMHEAPALLOC` in `ThreadSectionParser`)?

Reply:
I wanted to return information from those lines so declared those tags. Having declared those tags I need to parse them, otherwise the lookahead might not be sufficient to ignore them. However, the DTFJ API doesn't have a way to return the information from JavaThread. Would it be appropriate to return the Java characteristics of the ID, daemon flag, and heap allocation bytes from ImageThread.getProperties()?

Reply:
Capturing additional information from java dumps is something that should be done in a separate pull request if we think it's worthwhile.